tests/pom.xml

```diff
@@ -612,6 +615,9 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     </assembly>
                   </build>
                   <run>
+                    <dependsOn>
+                      <container>artemis</container>
+                    </dependsOn>
```

Comment:
why do you think that we need these?

Reply:
I ran into the problem that the device registry is started before the qdrouter which leads to the device registry startup failing (as it requires now a connection to the AMQP network) and consequently the integration tests fail.

Since we use custom networks I thought this is the only option to define startup order:

> To express start order dependencies using custom networks refer to the dependsOn configuration.

(see http://dmp.fabric8.io/#start-depends-on)

Did I miss something?

Reply:
Also I think if we properly set all dependencies we could set `<startParallel>` to true which would make the integration tests run faster.

Reply:
> The containers are started in the order that they are defined in the pom.xml. In order to start the registry after the router, simply move the router container definition before the registry's.

Reply:
> Also I think if we properly set all dependencies we could set <startParallel> to true which would make the integration tests run faster.

We did that at the beginning of our journey but quickly realized that it is close to impossible to understand the logs in case something goes wrong during startup. It also does not speed up the startup (on Travis) because the containers then compete for the (scarce) CPU resources and some containers may even fail to start up because they run into the configured timeout ...

Reply:
Okay, thanks for clarification. I will change the order the POM then.